### PR TITLE
[fix] Improve DB performance

### PR DESCRIPTION
**Pull request by Laumesis**

### CODE REVIEW

File: Database/QuestieDB.lua [hunk 1]
```diff
@@ -832,39 +811,26 @@ function QuestieDB:GetQuest(questId) -- /dump QuestieDB:GetQuest(867)
...
-            end
-        end
-    end
-
-    local zoneOrSort = rawdata[questKeys.zoneOrSort]
-    if zoneOrSort and zoneOrSort ~= 0 then
-        if zoneOrSort > 0 then
-            QO.Zone = zoneOrSort
-        else
-            QO.Sort = -zoneOrSort
```
Comment (BreakBB) [inline Database/QuestieDB.lua:853]: I guess QO.Sort was never used so you removed it?

File: Database/QuestieDB.lua [hunk 2]
```diff
@@ -935,34 +902,37 @@ function QuestieDB:GetCreatureLevels(quest)
     return creatureLevels
 end
 
-local playerFaction = UnitFactionGroup("player")
+local playerFaction = UnitFactionGroup("player")  -- TODO: FIXME: Api call won't return correct value yet. Only after player entering world event
```
Comment (BreakBB) [inline Database/QuestieDB.lua:905]: Don't forget this TODO 😁 Can't you simply use QuestiePlayer for this?

**Review by BreakBB — COMMENTED** [review]
I know this is still a draft, but I find these changes quite readable and couldn't find much to argue 😁 👍🏻

File: Database/QuestieDB.lua (hunk 1 above)
Comment (Laumesis) [reply Database/QuestieDB.lua:853]: Both `.Zone` and `.Sort` are never used

File: Database/QuestieDB.lua (hunk 2 above)
Comment (Laumesis) [reply Database/QuestieDB.lua:905]: It is a bug in old old old master, I just mentioned it to be a bug 😛

**Comment by Laumesis** [comment]
```
git rebase master
git push --force-with-lease
```

**Pull request closed without merging**